clang/include/clang/CIR/Dialect/Passes.td

```diff
@@ -75,6 +75,20 @@ def LoweringPrepare : Pass<"cir-lowering-prepare"> {
   let dependentDialects = ["cir::CIRDialect"];
 }
 
+def StructuredCFG : Pass<"cir-structured-cfg"> {
```

Comment:
I vote for `cir-flatten-cfg`, otherwise we're given the impression that we are building a structured CFG, but we are doing the opposite, going from structured to flat basic block control flow.